src/Skeleton/Skeleton.css

```diff
@@ -0,0 +1,30 @@
+@keyframes progress {
+    0% {
+        /* background-position: -200px 0; */
```

Comment:
Did you intend to keep the comments in this file?

Reply:
yes, this is from an external library, and this code works in every browser apart from edge, hopefully we will be able to enable it one day.